src/test/regress/expected/citus_depended_object.out

```diff
@@ -0,0 +1,183 @@
+-- PG_CLASS VISIBILITY
+-- check if we correctly determine whether a relation is citus dependent or not.
```

Comment:
we typically follow this structure to avoid forgetting any objects:

```SQL
CREATE SCHEMA citus_dependend_object;
SET search_path TO citus_dependend_object;


....

DROP SCHEMA citus_dependend_object CASCADE;
```